src/cf.data.pre

```diff
@@ -1422,6 +1422,10 @@ endif
 	  # adaptation_meta because it starts matching immediately after
 	  # the service has been selected for adaptation.
 
+	acl aclname adaptation_rep_header header-name [-i] any\.regex\.here
```

Comment:
Does this new ACL match adaptation response headers delivered in

1. An error response from the adaptation service (e.g., ICAP 500)?
2. An ICAP 204 (Leave As Is) response (or equivalent)?